_includes/about.html

```diff
@@ -1,20 +1,19 @@
-    <!-- About Section -->
-    <section id="about" class="container content-section">
-        <div class="row">
-            <div class="col-lg-6 columns">
-                <h3 id="new-members">New To Code For Nashville? Visit our <a href="/new-members/">New Members</a> page.</h3>
-
-                <h2 class="text-center">About Code For Nashville</h2>
-                <p>Code for Nashville is Nashville's new <a href="http://www.codeforamerica.org/">Code for America Brigade</a>. We are volunteers who collaborate with Metro government to promote <a href="https://en.wikipedia.org/wiki/Open_data">open data</a> and build <a href="https://en.wikipedia.org/wiki/Civic_application">civic apps</a>.</p>
-
-                <p>We are a growing organization and need members with all types of skills. </p>
-
-                <p>The best way to become involved is to attend our next <a href="http://www.meetup.com/code-for-nashville/">hack night</a>. We meet monthly at the Sevier Park Community Center.</p>
-
-            </div>
-            <div class="mailing-list-link col-lg-6 columns text-center">
-                <link href="https://app.e2ma.net/css/signup.med.css" rel="stylesheet" type="text/css"><script type="text/javascript" src="https://app.e2ma.net/app2/audience/tts_signup/1797288/3a81fd3be022562747163cec7bab201c/1759497/"></script><div id="load_check" class="signup_form_message" >This form needs Javascript to display, which your browser doesn't support. <a href="https://app.e2ma.net/app2/audience/signup/1797288/1759497/"> Sign up here</a> instead </div><script type="text/javascript">signupFormObj.drawForm();</script>
-            </div>
+<div class="new-members">
+    <p>New To Code For Nashville? Visit our <a href="/new-members/">New Members page</a>.</p>
+</div>
+<section id="about">
+    <div class="about-cfn">
+        <h2>About</h2>
+        <p>Code for Nashville is Nashville's new <a href="http://www.codeforamerica.org/">Code for America Brigade</a>. We are volunteers who collaborate with Metro government to promote <a href="https://en.wikipedia.org/wiki/Open_data">open data</a> and build <a href="https://en.wikipedia.org/wiki/Civic_application">civic apps</a>.</p>
+        <p>We are a growing organization and need members with all types of skills.</p>
+        <p>The best way to become involved is to attend our next <a href="http://www.meetup.com/code-for-nashville/">hack night</a>. We meet monthly at the Sevier Park Community Center.</p>
```

Comment:
the last sentence needs to be removed as we don't meet at sevier park anymore - i think linking to meetup is the best option for now while we are still structuring our meetups